rj_msgs/msg/RobotStatus.msg

```diff
@@ -1,3 +1,5 @@
+builtin_interfaces/Time timestamp
```

Comment:
I'm slightly curious as to why this is added now, but not earlier. I noticed at one point that most of our messages don't use anything like std_msgs/Header.msg so we don't have that information for most messages. Do we have to consider the possibility of out-of-date or out-of-order message delivery on the gameplay side or otherwise?